canonical-web-and-design/ubuntu.com
# Advantage shop confirmation

# Done

Added information regarding a new subscription to /advantage

# QA
- Visit /advantage/subscribe, and sign in with SSO if you are prompted to
- Make your product selection, add it to the cart and click "Buy now"
- Fill out the payment details
  - Fill out all card fields by tapping the 4 & 2 keys until you can't input anything any more
    - (you'll end up with card number: 4242424242424242, expiry: 04/24, cvc: 242 and zip: 42424)
  - Address fields can be anything
- Click "Continue", accept Ts&Cs, then click "Pay"
- The payment should be successful, and when it completes you'll be redirected to /advantage
- Your new subscription should be easily visible on the page, with a "New" badge, the "Attached" tab should be expanded and include a welcome message along with instruction on attaching your machines.
- There should also be a "Where?" button (in reference to configuring services), and clicking that should trigger an animated icon on tabs for the services your subscription includes.
